apps/every/bos.config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "appAccount": "every.near",
```

Comment:
I've spent some time looking into the every app and it's even more intriguing than last time. Here are my thoughts:

- The `thing` tab: Personally as a developer I prefer it like this than a form input, it helps me understand what I'm creating better, and I can see how it will be great with custom creators too, like if you select efiz.near/type/doc you get the editor from Create. 
- The `template` tab: I like how the props are automatically supplied, that's really good. We definitely need that for creating template for create too.
- The `plugin` tab: this is still a strange concept to me, but I think I'm starting to get it after seeing this, I think it's more focused on handling the data, like pass in timestamp and receive a human readable date? I have a feeling this would work great with Create too, but Idk how it can be implemented. If you have any ideas let me know.

- My last point is that I see the concept of `everything` having 2 access points. For developers, where you have access to every thing and type and template and plugin and we can create with it as we want, it's like an innovative type of Database Manager. And for normal users, they should have access to one thing/type/template at a time, there should be full apps integrating the everything concept behind the scenes to simplify it for users. This app represents the access point for developers and will even be greater after integrating custom thing creators like the old one, and `create` is one of the examples for a normal user that simplify the process for them. I will be glad to contribute to this experiment too after finishing create.

Reply:
Awesome, I'm glad you like this.

Regarding plugins, yes exactly they are focused on handling data. I'm still trying to figure them out too, but we can take a modular approach like how they do in [Plate.js](https://platejs.org/). Plate.js is a rich text editor, and then bold is a plugin that takes your regular text and adds markdown pattern to bold it. Other examples of plugins may be an editor (so you can modify the thing), a validator (to validate data against a type), a normalizer (to normalize data into a type), a resolver (if only a reference to external data is stored and so we need to resolve it). But again, still figuring it out, will think about Create more.

> I will be glad to contribute to this experiment too after finishing create.

100% dawg! This is just the beginning! I totally align with you on this view. BOS Hacks in right after this hackathon, we can dive right into it and just keep building.